### PR TITLE
Create missing entities

### PR DESCRIPTION
**Pull request by Sc3l3t0n**

### CODE REVIEW

File: src/TuDa.CIMS.Api/Database/CIMSDbContext.cs [hunk 1]
```diff
@@ -5,14 +5,35 @@ namespace TuDa.CIMS.Api.Database;
...
+
     public DbSet<AssetItem> AssetItems { get; init; }
     public DbSet<Consumable> Consumables { get; init; }
     public DbSet<Substance> Substances { get; init; }
     public DbSet<GasCylinder> GasCylinders { get; set; }
     public DbSet<Chemical> Chemicals { get; init; }
     public DbSet<Solvent> Solvents { get; set; }
+
+    #endregion
+
```
Comment (devidsdt) [inline src/TuDa.CIMS.Api/Database/CIMSDbContext.cs:18]: What is #region for?

File: src/TuDa.CIMS.Shared/Dtos/UpdateSubstanceDto.cs [hunk 2]
```diff
@@ -10,7 +11,7 @@ namespace TuDa.CIMS.Shared.Dtos;
 public abstract record UpdateSubstanceDto : UpdateAssetItemDto
 {
     /// <summary>
-    /// An unique identifier for the chemical.
```
Comment (devidsdt) [inline src/TuDa.CIMS.Shared/Dtos/UpdateSubstanceDto.cs:13]: I think "an unique" was correct

**Review by devidsdt — APPROVED** [review]

File: src/TuDa.CIMS.Api/Database/CIMSDbContext.cs (hunk 1 above)
Comment (Sc3l3t0n) [reply src/TuDa.CIMS.Api/Database/CIMSDbContext.cs:18]: Region is for the IDE, so you can collapse them and the Context is not so cluttered

File: src/TuDa.CIMS.Shared/Dtos/UpdateSubstanceDto.cs (hunk 2 above)
Comment (Sc3l3t0n) [reply src/TuDa.CIMS.Shared/Dtos/UpdateSubstanceDto.cs:13]: I thought so too, but the IDE said it's correct like that. The web says it too

**Pull request merged**